std_msgs/Header header
string mission_id
sensor_msgs/Image frame
bar_detector/LineData[] left_vbar
bar_detector/LineData[] right_vbar
bar_detector/LineData[] left_hbar
bar_detector/LineData[] right_hbar

